examples/clusterscoped/csib-3-exclude-names.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: intent.security.nimbus.com/v1alpha1
+kind: SecurityIntent
+metadata:
+  name: escape-to-host
+spec:
+  intent:
+    id: escapeToHost
+    description: "A attacker can breach container boundaries and can gain access to the host machine"
+    action: Block
+---
+apiVersion: intent.security.nimbus.com/v1alpha1
+kind: ClusterSecurityIntentBinding
+metadata:
+  name: escape-to-host-binding
+spec:
+  intents:
+    - name: escape-to-host
+  selector:
+    nsSelector:
+      excludeNames:
+        - ns-2
+        - ns-3
```

Comment:
My reasoning was that either you would have a matchNames or an excludeNames. So, excludeNames means include everything else except the names mentioned in excludeNames. So, keeping it this way for now. In case we see a usecase which is not satisfied by this approach, we can take a relook

Reply:
It could lead to some ambiguity and I believe explicit is better than implicit to avoid ambiguity.

Reply:
Leaving this as it is for now. Either you have a matchName or excludeName. Not both.